src/app/api/events/route.ts

```diff
@@ -10,14 +10,14 @@ export async function GET(request: Request) {
   const supabase = createServerClient(cookieStore)
 
   const url = new URL(request.url)
-  const creatorId = url.searchParams.get('creatorId')
-  const eventId = url.searchParams.get('eventId')
+  const creatorid = url.searchParams.get('creatorid')
```

Comment:
I believe convention was camel case so I'm not sure why this is being changed away from it.  Any particular reason? There seems to be no functional change except now searchParams is forced to go lower case

Reply:
I think I ran into consistency issues where some of them were "eventId" and some were "eventid" and I wasn't sure if the issues I was running into were because the DB was changing them to lowercase, but I will change them back to "eventId" and see if it works.

Reply:
the db calls it 'creator' and 'id' so shouldn't be a problem.

Reply:
Changed back to creatorId and eventId everywhere.